### DISCUSSION TITLE
Nominating Joshua Casey for Python Contributor role

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#141] [Python Team]
I'd like to nominate @joshuatcasey as a contributor for the python sub team. He has made multiple valuable contributions - both in terms of code and in terms of architectural discussions and decision-making.

cc @paketo-buildpacks/python-maintainers 

Thanks,
Rob

**Comment by ryanmoran** [comment]
Congrats @joshuatcasey! You've been added to the Python Contributors team.

**Comment by joshuatcasey** [comment]
Thanks!
